### ISSUE TITLE
The all verification has been failed using zksolc 1.4.0

### ISSUE THREAD

**Issue by wshino** [matter-labs/hardhat-zksync#847]
### 🐛 Bug Report for hardaht-zksync plugins

When I deploy some contracts with verification, the contract has been deployed successfully, but verification failed.
I never seen this error before I was using older zksolc versions.

```
@m-ikowit contracts % npx hardhat deploy-zksync --script deploy.ts --network zkSyncSepoliaTestnet

Starting deployment process of "TestERC20"...
Estimated deployment cost: 0.0001551017 ETH

"TestERC20" was successfully deployed:
 - Contract address: 0xe1eCC6FD2dfD612BE600d3b5258C1dCc8e681533
 - Contract source: src/zksync-deploy/TestERC20.sol:TestERC20
 - Encoded constructor arguments: 0x00000000000000000000000000000000000000000000000000000000000000400000000000000000000000000000000000000000000000000000000000000080000000000000000000000000000000000000000000000000000000000000000954657374546f6b656e000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000025454000000000000000000000000000000000000000000000000000000000000

Requesting contract verification...
Your verification ID is: 5132
Error in plugin @matterlabs/hardhat-zksync-verify: Backend verification error: Deployed bytecode is not equal to generated one from given source
```

Network configuration is here:

```
networks: {
    zkSyncSepoliaTestnet: {
      url: "https://zksync-era-sepolia.blockpi.network/v1/rpc/public",
      ethNetwork: "sepolia",
      zksync: true,
      verifyURL: "https://explorer.sepolia.era.zksync.dev/contract_verification",      
    },
}
```

#### 💥 Plugin name

`@matterlabs/hardhat-zksync-verify`

#### 📝 Description

Provide a clear and concise description of the bug.

#### 🔄 Reproduction Steps

1. Run `npx hardhat deploy-zksync`

#### 🤔 Expected Behavior

Describe what you expected to happen.

#### 😯 Current Behavior

Describe what actually happened.

#### 🖥️ Environment

zksolc: 1.4.0

#### 📋 Additional Context

Add any other context about the problem here. If applicable, add screenshots to help explain.

#### 📎 Log Output

```
Paste any relevant log output here.
```

**Comment by kiriyaga-txfusion** [comment]
We are in the process of fixing the issue. Issue appeared at verify BE. In the meantime, adjust your Hardhat configuration to support either Solidity version 0.8.19 or switch zksolc version to 1.3.23 as a temporary solution.

**Comment by githubdoramon** [comment]
Apparently this is now fixed with 1.4.0 being available at backend
